src/js/common/schemaform/SaveInProgressIntro.jsx

```diff
@@ -37,7 +37,7 @@ export default class SaveInProgressIntro extends React.Component {
           <div>
             <div className="usa-alert usa-alert-info schemaform-sip-alert">
               <div className="usa-alert-body">
-                You can save this form in progress, and come back later to finish filling it out.
+                <strong>Note:</strong> As you are signed in to your account, we can prefill part of your application based on your account details. You can also save your form in progress, and come back later to finish filling it out.
```

Comment:
@peggygannon I made a slight tweak for if a user is logged in but doesn't yet have a saved form. Please let me know if additional changes should be made here.